pkg/cmd/kind/load/image-archive/image-archive.go

```diff
@@ -49,7 +49,7 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 			}
 			return nil
 		},
-		Use:   "image-archive",
+		Use:   "image-archive /path/to/tar-archive-file",
 		Short: "Loads docker image from archive into nodes",
```

Comment:
i think it would be better to put `IMAGE-PATH` or something more consistent with the other command but /shrug

Reply:
it sounds good, i have changed it to <IMAGE.tar>[1]
[1]: <> is a [Unix-style](https://en.wikipedia.org/wiki/Usage_messag)